config/secrets.yml

```diff
@@ -1,8 +1,8 @@
 development:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: '3a954ea76b427c8680c8d2652ab5f7c9bfec80e391f947e913595c228f3fc93000996740df946c941ecbd02f1a6e094b405f548ba5eceeb1159b3ee2ee88e75c'
 
 test:
```

Comment:
Using windows, i need to do this to run vagrant provision as it doesn't work on dummyToken.sh. This mightaffect the production side and have some security issue. @yhchong how should I handle this?

Reply:
Removed files from commit